myTrip/tests/checkpoint/unit_test/test.py

```diff
@@ -0,0 +1,178 @@
+"""Testing module for checkpoint views."""
+
+import json
+from datetime import datetime
```

Comment:
json and datetime are standart python libs, so after them should be space, before djangos imports